src/CommunityToolkit.Maui.UnitTests/Converters/IsInRangeConverterTests.cs

```diff
@@ -8,6 +8,8 @@ public class IsInRangeConverterTests : BaseOneWayConverterTest<IsInRangeConverte
 {
 	public const string FalseTestObject = nameof(FalseTestObject);
 	public const string TrueTestObject = nameof(TrueTestObject);
+	public const DateTimeKind LocalDateTimeKind = DateTimeKind.Utc;
```

Comment:
should be Local?

Reply:
You are right.  I have re-named the variable to `UtcDateTimeKind`.